nixos/modules/system/boot/uvesafb.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.boot.uvesafb;
+in {
+  options = {
+    boot.uvesafb = {
+      enable = lib.mkEnableOption (lib.mdDoc "uvesafb");
+
+      gfx-mode = lib.mkOption {
+        type = lib.types.str;
+
+        default = "1024x768-32";
+
+        description = lib.mdDoc "Screen resolution";
```

Comment:
Please add a few examples.

Reply:
I linked the kernel documentation about it